_README.md

```diff
@@ -44,6 +44,14 @@ bower install
 
 ### Roadmap
 
+1. Access Microphone
+1. Grab data from microphone
+1. Transmit Data
+1. Server Maintains Connection
+1. Passes through server
```

Comment:
Now that we've settled on WebRTC, should clarify that the audio will stream client-to-client (as peers) and the node server will handle the signaling required to set up the peer to peer connections.